docs/source/installation.rst

```diff
@@ -4,20 +4,66 @@ Installation
 Requirements
 ------------
 
-Before you can start using SchemaSpy you must have installed two things in your system environment.
+Before you can use SchemaSpy you must have the following requirements available on your local system.
 
-Java 8
+* Java
+* a JDBC driver file for your specific database engine 
+* viz.js or Graphviz
+
+Java
+^^^^^^
+
+You will need to have a supported version of Java installed, which includes:
+
+* Java version 8 or higher
+* OR OpenJDK 1.8 or higher - an open-source alternative
+
+You can run ``java -version`` in a terminal to check the version of any currently installed Java.
+
+If you don't already have a proper version of Java installed, see `OpenJDK <https://openjdk.java.net/install/>`_ or `Oracle Java <https://www.oracle.com/java/technologies/javase-downloads.html>`_ for download and install instructions for your operating systems.
+
+
+JDBC Driver
 ^^^^^^
```

Comment:
Title underline too short.

Reply:
Increased the underlining in the latest commit.